pkg/agent/tunnelserver/tunnelserver.go

```diff
@@ -264,6 +266,17 @@ func (t *tunnelServer) LoftConfig(ctx context.Context, message *tunnel.Message)
 	return &tunnel.Message{Message: string(out)}, nil
 }
 
+func (t *tunnelServer) GPGPublicKeys(ctx context.Context, message *tunnel.Message) (*tunnel.Message, error) {
```

Comment:
docblock explaining what this function is doing would be nice

Reply:
Let's do that another time when we document all of the GRPC endpoints :)